sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -83,15 +83,15 @@
 
     <app-header slot="header" fixed shadow>
       <app-toolbar>
-        <div main-title class="title">
+        <div id="title" main-title class="toolbar-title">
           <!--TODO: @yebrahim replace this with SVG version of the logo-->
-          <img src="images/datalab-logo.png" class="title-image">
+          <img src="/images/datalab-logo.png" class="title-image">
```

Comment:
The reason for this change is because the tests run from a different path than the actual elements, so if the elements include relative paths, those paths won't be resolved in the test.
Maybe we should have a discussion on the base path change, I want to understand more to see if there's a different way to do it with the new UI.